x-pack/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/CreateEditCustomLinkFlyout/link_preview.tsx

```diff
@@ -66,43 +65,52 @@ export function LinkPreview({ label, url, filters }: Props) {
   const { formattedUrl, error } = replaceTemplateVariables(url, transaction);
 
   return (
-    <EuiPanel betaBadgeLabel="Preview" paddingSize="l">
-      <EuiText
-        size="s"
-        color={getTextColor(label)}
-        className="eui-textBreakWord"
-        data-test-subj="preview-label"
-      >
-        {label
-          ? label
-          : i18n.translate(
-              'xpack.apm.settings.customizeUI.customLink.default.label',
-              { defaultMessage: 'Elastic.co' }
-            )}
-      </EuiText>
-
-      <EuiText
-        size="s"
-        color={getTextColor(url)}
-        className="eui-textBreakWord"
-        data-test-subj="preview-url"
-      >
-        {url ? (
-          <EuiLink
-            href={formattedUrl}
-            target="_blank"
-            data-test-subj="preview-link"
-          >
-            {formattedUrl}
-          </EuiLink>
-        ) : (
-          i18n.translate(
-            'xpack.apm.settings.customizeUI.customLink.default.url',
-            { defaultMessage: 'https://www.elastic.co' }
-          )
-        )}
-      </EuiText>
-      <EuiSpacer />
+    <EuiCard
```

Comment:
I think this conversion if forcing EuiCard to be something that it is not. My honest suggestion would be to just move the "Preview" text to a heading similar to the rest of the sections with: `<EuiTitle size="xs"><h3 /></EuiTitle>` before the panel.